tooling/cli/lib/deploy.js

```diff
@@ -196,31 +195,42 @@ async function triggerBatch(instance, key) {
 }
 
 async function getHeliumUploadData(instance) {
-  const endpoint = instanceEndpoint(instance);
-
-  const options = {
-    method: 'POST',
-    headers: { 'Content-Type': 'application/json' },
-    body: JSON.stringify({ query: KEY_QUERY, variables: { nickname: instance.nickname } })
-  };
-
-  let responseData = {};
-  const launchData = await fetch(endpoint, options).then(r => r.json());
-
-  if (launchData && launchData.data) {
-    const {
-      data: {
-        HeliumLaunchData: { key, signedUrl }
-      }
-    } = launchData;
+  return new Promise((resolve, reject) => {
+    const endpoint = instanceEndpoint(instance);
 
-    responseData = {
-      key,
-      signedUrl
+    const options = {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json' },
+      body: JSON.stringify({ query: KEY_QUERY, variables: { nickname: instance.nickname } })
     };
-  }
 
-  return responseData;
+    fetch(endpoint, options)
+      .then(r => r.json())
+      .then(res => {
+        const resObj = res;
+        const responseData = {};
```

Comment:
Sorry wasn't clear; meant you could just do

```javascript
const responseData = {
  key,
  signedUrl
}
```

on L220. Think current implementation would cause an error attempting to redefine `const`